pkg/pulumiyaml/run_options_test.go

```diff
@@ -106,6 +106,62 @@ resources:
 	}
 	err := pulumi.RunErr(func(ctx *pulumi.Context) error {
 		runner := newRunner(ctx, template, newMockPackageMap())
+		runner.setDefaultProviders()
```

Comment:
Use `requireNoErrors` or the appropriate `assert.` method to assert the return value isn't an error.